rebar.config

```diff
@@ -1,106 +1,125 @@
-%%% -*-mode:erlang;coding:utf-8;tabidth,:4;c-basic-offset:4;indent-tabs-mode:()-*-
-%%% ex: set ft=erlang fenc=utf-8 sts=4 ts=4 sw=4 et:
-%%%
+%% == Erlang Compiler ==
 
-%%% Require OTP 19.1 at a bare minimum
-{minimum_otp_vsn, "19.1"}.
+{minimum_otp_vsn, "22"}.
 
-%% Plugins
-{plugins, [rebar3_hex]}.
-{shell, [
-    {apps, [lager, sync, gen_rpc]}
-]}.
-
-{erl_opts, [debug_info,
-    {warn_format, 1},
```

Comment:
Most of this is moving stuff around, but some (default) options were actually removed to save on maintenance costs.